### ISSUE TITLE
Import tool not working.

### ISSUE THREAD

**Issue by Tucker2343** [cynicaloptimist/improved-initiative#572]
Last night I was attempting to import stat blocks from the new books, and I kept running into this error.
![50F9DAF1-3872-4E62-AE49-9F0C9E559FAF](https://user-images.githubusercontent.com/15664374/142731800-aeb6781e-e097-4f0a-83a2-d21b64208574.png)

**Comment by cynicaloptimist** [comment]
Interesting, looks like the statblock payload may be too large on some of them- which statblocks is this happening for?

**Comment by Tucker2343** [comment]
I was trying to import the stat block for an Ancient Dragon Turtle.

**Comment by cynicaloptimist** [comment]
Yep, looks like this guy has a humongous Description section. Try turning off "Include description" on the extension settings. I'll see if I can have the extension detect when the Statblock is too big.

**Comment by Tucker2343** [comment]
Sorry for the late reply, been busy with the holidays. I will give this a try and let you know.
